parse_roots.go

```diff
@@ -0,0 +1,22 @@
+// +build !windows
+
+package main
+
+import (
+	"crypto/x509"
+	"github.com/pkg/errors"
+)
+
+func parseRoots(roots *x509.CertPool ) error{
+	roots, err := x509.SystemCertPool()
+	if err != nil {
+		return errors.Wrap(err, "Failed to parse root store")
+	}
+
+	for _, ident := range idents {
```

Comment:
I think we should also continue to fallback to `gocertifi.CACerts()` if we cannot open the `SystemCertPool` on macOS / Linux.

While the fallback path was primarily there for Windows support, macOS could still have a non-nil `err` if opening the root store failed.

Reply:
Sounds good, I’ll go ahead and get to work on implementing this